src/donor_dashboard/templates/donor_dashboard/list.html

```diff
@@ -169,4 +169,21 @@ <h2 class="subtitle mt-5">
         </div>
         </div>
     </div>
+    <script>
```

Comment:
Can you please shift this to a new Javascript file under static?

Reply:
I think this has been done in develop! Mine is just outdated; I'll remove the snippet.

Reply:
Sure, thanks

Reply:
@arsalananwar11 If all looks good could you resolve the requested changes? I'll merge this one and continue refactoring in my other PR!